.npmignore

```diff
@@ -6,12 +6,17 @@
 .vagrant
 report.tap
 build/
+coverage/
 demos/
 examples/
+instrumented/
 products/
+reports/
 tests/component-tests/
+tests/configs/
 tests/framework-tests/
 tests/manual-tests/
 tests/node-tests/
 tests/test-core/testTests/
 tests/all-tests.html
+tests/testem.js
```

Comment:
This is not test output, but a crucial file required to run the browser tests.

Reply:
It replaced the previous testem.json file.

Reply:
@the-t-in-rtf thanks for looking over this. The ignores here and the previous one you mentioned for the configs are only for the npm package. I figure that testing should have been completed before publishing and the only things we typically publish from the test directory are the infrastructure pieces that are needed for others to write tests. Also, these files haven't been added to the gitignore so they will still show up in the git repo.